packages/strip/README.md

```diff
@@ -36,59 +36,88 @@ export default {
     dir: 'output',
     format: 'cjs'
   },
-  plugins: [
-    strip({
-      labels: ['unittest']
-    })
-  ]
+  plugins: [strip({})]
 };
 ```
 
 Then call `rollup` either via the [CLI](https://www.rollupjs.org/guide/en/#command-line-reference) or the [API](https://www.rollupjs.org/guide/en/#javascript-api).
 
+### Default Options
+
+By default the **strip** plugin applies the following options, you can override by specifing options manually.
+
+> consider files matching **'\*\*/\*.js'**  
+> remove functions matching '**console.\***' and '**assert.\***'  
+> remove any **debugger** statements
+
+### Custom Options Example
+
+```js
+plugins: [
+  strip({
+    include: '**/*.(mjs|js)',
+    exclude: 'tests/**/*',
+    debugger: true,
+    functions: ['console.log', 'assert.*'],
+    labels: ['unittest'],
+    sourceMap: true
+  })
+];
+```
+
 ## Options
 
 ### `include`
 
 Type: `String | RegExp | Array[...String|RegExp]`<br>
-Default: `['**/*.js']`
+Default: `['**/*.js']`<br>
+Example: `include: '**/*.(mjs|js)',`<br>
 
 A pattern, or array of patterns, which specify the files in the build the plugin should operate on.
 
 ### `exclude`
 
 Type: `String | RegExp | Array[...String|RegExp]`<br>
-Default: `[]`
+Default: `[]`<br>
+Example: `exlude: 'tests/**/*',`<br>
 
 A pattern, or array of patterns, which specify the files in the build the plugin should _ignore_.
 
 ### `debugger`
 
 Type: `Boolean`<br>
-Default: `true`
+Default: `true`<br>
+Example: `debugger: false,`<br>
 
-If `true`, instructs the plugin to remove debugger statements.
+If `true` or unspecified, instructs the plugin to remove debugger statements.
```

Comment:
`or unspecified` is not not necessary and doesn't add any value here, please revert.

Reply:
ok